Ct 1785 edit ico closure

beccapearce opened the pull request.

aliuk2012 commented on app/services/notify_responder_service.rb:
```diff
@@ -0,0 +1,24 @@
+      .deliver_later
+  end
+
+  def ready_to_send
```
do we need this method?

Commit:
CT-1785 testing overturned ICO appeals

aliuk2012 reviewed (dismissed):
This PR seems to introduce `config/state_machine/moj.yml`

Commit:
CT-1918 ICO appeal case details page copy Changes

use translate for case method on request
other copy changes in translation file

stephenrichards committed:
CT-1922 Copy ICO office name from original appeal

This is implementd as a before_validate callback on
Case::Overturned::Base which copies the ICO officer name
from the original appeal

There is also validate to ensure that it is present because
it is possible that the user can edit the ICO officer name and
remove it.

stephenrichards committed:
CT-1919 Case::Base subtypes specify which state machine config to use

beccapearce committed:
Merge branch 'master' into ct-1785-edit-ico-close

beccapearce committed:
Merge branch 'master' into ct-1785-edit-ico-close
